docs/dev.md

```diff
@@ -62,3 +64,22 @@ Integration tests can read the following system properties to obtain information
 | liberty.hostname | The host name of the Liberty server. |
 | liberty.http.port | The port used for client HTTP requests. |
 | liberty.https.port | The port used for client HTTP requests secured with SSL (https). |
+
+##### devc, Container Mode
+
+This is a technology preview.
+
+Start a Liberty server in a local container using the Docker file that you provide. The Docker file must copy the application .war file and the server configuration files that the application requires into the container. An alternative to the `devc` goal is to specify the `dev` goal with the `-Dcontainer` option. A sample Docker file is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the docker file.
```

Comment:
Throughout, use "Dockerfile" instead of "Docker file"

Reply:
👌